crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -445,10 +490,34 @@ where
                 .contains(MeshPipelineKey::MOTION_VECTOR_PREPASS)
                 .then_some(ColorTargetState {
                     format: MOTION_VECTOR_PREPASS_FORMAT,
-                    blend: Some(BlendState::REPLACE),
+                    // BlendState::REPLACE is not needed here, and None will be potentially much faster in some cases.
+                    blend: None,
                     write_mask: ColorWrites::ALL,
                 }),
         );
+
+        if key.mesh_key.contains(MeshPipelineKey::DEFERRED_PREPASS) {
+            targets.push(
+                key.mesh_key
```

Comment:
i think you should remove the outer condition here since the check is done inside already.

the point of pushing None if the key is not set, is so that the target locations don't shift around depending on what prepass outputs are requested. it doesn't matter currently since deferred outputs are last, but may matter in future.